app/models/concerns/media_oembed.rb

```diff
@@ -34,10 +34,12 @@ def get_oembed_data(original_url = nil, maxwidth = nil, maxheight= nil)
     url = original_url || self.url
     if Media.valid_raw_oembed?(self.data)
       self.data['oembed'] = self.data['raw']['oembed'].merge(width: maxwidth, height: maxheight, html: Media.default_oembed_html(url, maxwidth, maxheight))
+    elsif self.provider == 'file'
```

Comment:
Can this be one of the first `if`'s so we don't process the other cases? I'm just concerned that the other cases can require more HTML parsing, which doesn't make sense for files.

Reply:
sure!